pennylane_aqt/device.py

```diff
@@ -206,16 +221,8 @@ def _apply_operation(self, operation):
         device_wires = self.map_wires(operation.wires)
         device_wire_labels = device_wires.tolist()
 
-        name_parts = op_name.split(".")
-        if len(name_parts) == 1:
-            op_name = name_parts[0]
-            inv = False
-        elif len(name_parts) == 2 and name_parts[1] == "inv":
-            op_name = name_parts[0]
-            inv = True
-
         if op_name == "R":
```

Comment:
Does it cover the `RX`, `RY` and `RZ`?

Reply:
Yeah, I was puzzled by that for a bit too, its a little hidden. It's not `R`, that is some device-specific gate, I'm not sure of the specifics. 

It gets covered at the very end of `_apply_operation`. `RX`, `RY` and `RZ` are native to the device, so they don't need to be decomposed into anything. So for those, it gets the angle for rotation in line 213 and saves it as `par`. Then it gets through all of the special gate logic without doing anything, and then it reverses the sign on the parameter in the event the operation is `Adjoint` (previously if `inv`) (line 258),  and appends the operator to the queue using the `op_name` and `par`.

Reply:
Thanks!